README.md

```diff
@@ -11,6 +11,18 @@ language that recognizes Church encoded structures.
 Before doing anything useful with Morganey you need to install
 [sbt][scala-sbt] first.
 
+### Samples ###
+
+For language samples and examples please visit [docs/samples/][samples-dir] directory. To execute a sample run:
+
+    $ sbt "run <path-to-sample>"
+
+from the root of the source code directory. For example, to run the 15 first fibonacci numbers sample run:
+
+    $ sbt "run docs/samples/fib.mgn"
```

Comment:
I suggest to put such blocks into `console` environment such as:

    ```console
    $ sbt "run <path-to-sample>"
    ```

This way GitHub will color them more nicely.